lib/Conversation/CommentForm/CommentFormActions.js

```diff
@@ -1,31 +1,28 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Button from '../../Button';
-import MentionIcon from '../../../assets/icons/mention.svg';
 
 const CommentFormActions = props => (
   <div className="comment-form__actions">
-    <Button
-      clickHandler={() => {}}
-      types={['clear', 'size-sm']}
-      className="form__button"
-    >
-      <MentionIcon />
-    </Button>
-
     <span className="comment-form__actions comment-form__actions--submit">
       <Button types={['danger', 'size-sm']} clickHandler={props.onCancel}>
         Cancel
       </Button>
-      <Button types={['secondary', 'size-sm']} clickHandler={() => {}} isSubmit>
+      <Button
+        types={['secondary', 'size-sm']}
+        disabled={props.hasValue ? false : true}
```

Comment:
`!props.hasValue` would work just the same as this ternary operator. What do you think?